### PR TITLE
Add Shrinker as a script optimization technique

### PR DESCRIPTION
**Pull request by Geometer1729**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ We currently provide the following resources, broadly organized by topic.
 ### Script Optimization Techniques:
 * [Reducing Plutus Script Sizes](scriptsize.md)
 * [Optimizations to reduce CPU and Mem consumption](scriptmem.md)
+* [Shrinker](https://github.com/Geometer1729/Shrinker)
```
Comment (srid) [inline README.md]: Should we place this repo under github.com/Plutonomicon ?

cc @Fresheyeball @Benjmhart

Comment (Geometer1729) [reply README.md]: I think that makes sense. Are you able to add me to the Plutonomicon group so I can move it?

Comment (srid) [reply README.md]: @Geometer1729 It doesn't seem like I have permissions for that, but I've created https://github.com/Plutonomicon/Shrinker and invited you with Admin role.

Comment (Geometer1729) [reply README.md]: Thanks

**Review by srid — APPROVED** [review]

**Pull request merged**